cloudbuild.deploy.yaml

```diff
@@ -32,7 +32,10 @@ steps:
     cd server && \
     gcloud app deploy app_staging.yaml -q \
       --project=datcom-browser-staging \
-      --version=$SHORT_SHA
+      --version=$SHORT_SHA && \
+    gcloud app versions delete --project=datcom-browser-staging \
+    `gcloud app versions list --project=datcom-browser-staging | \
+    sed 's/  */:/g' | cut -f 2 -d : | tail -n +2 | head -1 | tr "\n" " "`
```

Comment:
maybe do the following updates:
tail -n +3 (or 4) so we keep more than 1 version around if there's only 2 versions remaining
head -n 1 for clarity

should we add a --sort-by to guarantee it's listed by ascending create date?

Reply:
Added number check and if check